.circleci/config.yml

```diff
@@ -36,6 +36,7 @@ parameters:
 
 jobs:
   check_latest_version:
+    resource_class: xlarge
```

Comment:
Wait, do we need this?

Reply:
Let's bump that only if strictly necessary, I'd say. Sad if it is, but it is what it is

Reply:
From slack:
I don't feel strongly about it (well, actually a little), but I like to use this an an indicator. I.e. the day it fails, we know something has changed
If we start with a huge box already, if at some point we introduce a change that doubles our foot print, we will never notice
But I could be convinced otherwise
Also, maybe adding a kurtosis clean -a after each test would not be terribly difficult and help a little bit

>>
merging it as it seems we're already at the limit, but will try to revisit next week